tensorboard/uploader/uploader_main.py

```diff
@@ -159,6 +159,16 @@ def _define_flags(parser):
         help="Experiment description. Markdown format.  Max 600 characters.",
     )
 
+    upload.add_argument(
```

Comment:
nit: Can we remove the blank line before this paragraph? I originally
misread this as a new top-level argument due to the spacing.

Reply:
Done.